datasets/HakaiWaterPropertiesInstrumentProfileResearch.xml

```diff
@@ -47,11 +47,12 @@
         <att name="subsetVariables">station_id, longitude, latitude</att>
     -->
     <addAttributes>
+        <att name="institution">Hakai Institute</att>
```

Comment:
would be a simpler merge if you maintain the order of the attributes as they were